### ISSUE TITLE
unused variable, false positive, DEFAULT/VALUE

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1218]
```abap
CLASS lcl_bar DEFINITION.
  PUBLIC SECTION.
    CONSTANTS const TYPE string VALUE 'value'.
    METHODS name
      IMPORTING iv_parameter TYPE string DEFAULT const.
ENDCLASS.
CLASS lcl_bar IMPLEMENTATION.
  METHOD name.
    WRITE iv_parameter.
  ENDMETHOD.
ENDCLASS.
```

**Comment by larshp** [comment]
hmm, a bit difficult right now

**Comment by larshp** [comment]
and

```abap
    CONSTANTS const TYPE string VALUE 'value'.
    DATA foo TYPE string VALUE const.
    WRITE foo.
```
